docs/content/docs/tutorials/sqs-example.md

```diff
@@ -0,0 +1,269 @@
+    clusters.
+  - [Helm 3.8+](https://helm.sh/docs/intro/install/) - A tool for installing and managing Kubernetes applications.
+
+### Install the ACK service controller for SQS
+
+Log into the Helm registry that stores the ACK charts:
+```bash
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+```
+
+Deploy the ACK service controller for Amazon SQS using the [sqs-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/sqs-chart). Resources should be created in the `us-east-1` region:
+
+```bash
+helm install --create-namespace -n ack-system oci://public.ecr.aws/aws-controllers-k8s/sqs-chart --version=v0.0.3 --generate-name --set=aws.region=us-east-1
+```
+
+For a full list of available values to the Helm chart, please [review the values.yaml file](https://github.com/aws-controllers-k8s/sqs-controller/blob/main/helm/values.yaml).
+
+### Configure IAM permissions
+
+Once the service controller is deployed, you will need to [configure the IAM permissions][irsa-permissions] for the
+controller to query the SQS API. For full details, please review the AWS Controllers for Kubernetes documentation for
+[how to configure the IAM permissions][irsa-permissions]. If you follow the examples in the documentation, use the value
+of `sqs` for `SERVICE`.
+
+## Create a SQS Queue
+
+Execute the following command to create a manifest for a basic SQS queue, with an inline policy with `SendMessage`
+permissions for the account owner, and submit this manifest to EKS cluster using kubectl.
+
+{{% hint type="info" title="Make sure environment variables are set" %}}
+If you followed the steps in the IAM permissions section above, the required environment variables `${AWS_REGION}` and
+`${AWS_ACCOUNT_ID}` are already set. Otherwise please set these variables before executing the following steps. 
+{{% /hint %}}
+
+```bash
+QUEUE_NAMESPACE=sqs-example
+QUEUE_NAME=basic-sqs
+
+kubectl create ns ${QUEUE_NAMESPACE}
+
+cat <<EOF > basic-sqs-queue.yaml
+apiVersion: sqs.services.k8s.aws/v1alpha1
+kind: Queue
+metadata:
+  name: ${QUEUE_NAME}
+  annotations:
+    services.k8s.aws/region: ${AWS_REGION}
+spec:
+  queueName: ${QUEUE_NAME}
+  policy: |
+    {
+      "Statement": [{
+        "Sid": "__owner_statement",
+        "Effect": "Allow",
+        "Principal": {
+          "AWS": "${AWS_ACCOUNT_ID}"
+        },
+        "Action": "sqs:SendMessage",
+        "Resource": "arn:aws:sqs:${AWS_REGION}:${AWS_ACCOUNT_ID}:${QUEUE_NAME}"
+      }]
+    }
+EOF
+
+kubectl -n ${QUEUE_NAMESPACE} create -f basic-sqs-queue.yaml
+```
+
+The output of above commands looks like
+
+```
+namespace/sqs-example created
+queue.sqs.services.k8s.aws/basic-sqs created
+```
+
+## Describe SQS Custom Resource
+
+View the SQS custom resource to retrieve the `Queue URL` in the `Status` field
+
+```bash
+kubectl -n $QUEUE_NAMESPACE describe queue $QUEUE_NAME
+
+Name:         basic-sqs
```

Comment:
Please split input and output into separate command blocks so users can copy the input using the "Copy to Clipboard" button

Reply:
+1